korangar/src/world/entity/mod.rs

```diff
@@ -67,6 +67,7 @@ pub enum EntityType {
 pub struct Common {
     pub entity_id: EntityId,
     pub job_id: usize,
+    pub hair_id: usize,
```

Comment:
Do entities besides the player also have a `hair_id`? I would assume it's only player characters, in wich case this should ot be in `Common`.